cachi2/core/package_managers/generic/models.py

```diff
@@ -10,7 +13,11 @@ class LockfileMetadata(BaseModel):
 
 
 class LockfileArtifact(BaseModel):
-    """Defines format of a single artifact in the lockfile."""
+    """
+    Defines format of a single artifact in the lockfile.
+
+    Only used for validation of the format of the lockfile.
+    """
```

Comment:
nitpick: I think it is worth adding a short description for all attributes in the class.

Reply:
Added